lib/InputOutputItemsUtility.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:io';
+import 'Item.dart';
+
+enum ItemTypes { raw, manufactured, imported }
+
+class InputOutputItemsUtility {
+  // list containing details of all items wanted
+  static List<Item> itemDetails = [];
+  static void inputItems() {
+    do {
+      // taking inputs of item name, item price, item type, item quantity
+      print("Enter item name");
+      String itemName = stdin.readLineSync()!;
+      print("Enter price of item");
+      double itemPrice = double.parse(stdin.readLineSync()!);
+      print(
+          "Enter 0 for raw type\nEnter 1 for manufactured type\nEnter 2 for " +
+              "imported type");
+      int itemTypeIndex = int.parse(stdin.readLineSync()!);
+      ItemTypes itemType = ItemTypes.values[itemTypeIndex];
+      print("Enter quantity of item");
+      double itemQuantity = double.parse(stdin.readLineSync()!);
+
+      // creating instance of type Item
+      Item newItem = new Item(itemName, itemType,
+          itemPrice: itemPrice, itemQuantity: itemQuantity);
+
+      // calculating tax for item
+      newItem.calculateTax();
+
+      // adding the item details to the list
+      itemDetails.add(newItem);
+
+      // checking if want another item
+      print("Do you want to enter details of any other item? Enter 'y' for" +
+          "YES and 'n' for NO (y/n):");
+    } while (stdin.readLineSync() == "y");
+
+    outputItems();
+  }
```

Comment:
inputItem() should be responsible only for input 
Do not do processing calls and output calls inside this 
Its a utility class.  Has set of utility functions that just performs a single task 
So for example, here inputItem() takes input and returns an object of type Item holding all the input values in its member variables